client/flutter/assets/content_bundles/protect_yourself.en.yaml

```diff
@@ -14,7 +14,7 @@ contents:
         <b>Cover your mouth and nose</b> with your bent elbow or tissue when you cough or sneeze.
     - image_name: protect-distance
       body_html: |
-        <b>Stay more than</b> 1 meter (3 feet) away from a person who is sick.
+        <b>Stay more than 1 meter</b> (3 feet) away from other people when possible.
```

Comment:
This is a change in medical content and should be reviewed by WHO @borowiecth

Reply:
Maintain at least a 1-metre distance between yourself and others. Maintain an even greater distance when indoors. 
Can be shortened to:

> Maintain at least a 1-metre distance with others and an even greater distance when indoors. 

https://www.who.int/emergencies/diseases/novel-coronavirus-2019/question-and-answers-hub/q-a-detail/q-a-on-covid-19-and-masks

Reply:
@borowiecth - I'd propose:

"**Distance at least a 1 meter** (3 feet) with others and an even greater distance when indoors"

This allows you to put emphasis on "Distance" and "1 meter" in a smaller bolded section whereas "Maintain" isn't so informative.

There's a separate question about "meter" vs. "metre". The WHO website uses both with slightly more usage of "metre". Around the internet in general, there's a 3X preference for "meter" when it comes to associations with COVID:
13M: "1 meter" covid - [Google Search]https://www.google.com/search?q=%221+meter%22+covid)
4M: "1 metre" covid - [Google Search]https://www.google.com/search?q=%221+metre%22+covid)